src/app/about-us/page.tsx

```diff
@@ -1,22 +1,32 @@
 import React, { FC } from "react";
 import AboutHero from "@/components/about-us/AboutHero";
-import AboutOurMission from "@/components/about-us/AboutOurMission";
-import BoardMembers from "@/components/about-us/BoardMembers";
-import MissionStatement from "@/components/about-us/MissionStatement";
-import Timeline from "@/components/about-us/Timeline";
-import missionStatementData from "@/data/about-us";
-import PageHead from "@/components/about-us/PageHeader";
+import { AboutOurMissionVision } from "@/components/about-us/AboutOurMissionVision";
+import { TimelineItem } from "@/components/about-us/Timeline";
+import { CallOutText } from "@/components/about-us/Callout";
+import { Flex, Separator } from "@radix-ui/themes";
 
 const AboutUsPage: FC = () => {
-  const { missionStatement, aboutOurMission, timeline } = missionStatementData;
-
   return (
-    <main className="">
+    <main>
+      <CallOutText message="some message here" color="green" />
       <AboutHero />
-      <MissionStatement missionStatement={missionStatement} />
-      <BoardMembers />
-      <AboutOurMission content={aboutOurMission} />
-      <Timeline items={timeline} />
+      <Flex
```

Comment:
**suggestion:** This feels like a useful component to refactor out (the flex wrapper and side-by-side title / text content).  I'd recommend making a component in `/src/components/text/SideBySide.tsx` and then calling it here.  The data can be set on this page and passed into the component.